src/Console/Command/ScopeHintCommand.php

```diff
@@ -54,15 +63,53 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $scopeValues = $this->scopeHintService->getConfigValuesForScopes($input->getArgument('config_path'));
+
+        $all_configs = $this->scopeHintService->getAllScopes();
+
+        // get all configurations
+        if($input->getOption('all')) {
+
+             $scopeValues = array();
+             foreach($all_configs as $element) {
+                 $result_values = $this->scopeHintService->getConfigValuesForScopes($element);
+                 foreach($result_values as $element2) {
```

Comment:
Please add a speaking name instead of `$element2`. I cannot figure out what `$element2` contains by just looking at this function.